app/javascript/CommentSubscription/__stories__/CommentSubscription.stories.jsx

```diff
@@ -0,0 +1,14 @@
+import { h } from 'preact';
```

Comment:
Is there any way to view this story while the PR is in a draft?

Reply:
If you grab the PR locally, you can run `yarn storybook`. I've also updated it so that you can view it here as well, https://devto-storybook.now.sh